### PR TITLE
Use kotl-kview

### PR DESCRIPTION
**Pull request by matsl**
# What

Use kotl-kview when changing label type from menu.

# Why

This was likely missed during refactoring when the prefix kotl was
added to the kview local variable.

### CODE REVIEW

File: kotl/kmenu.el [hunk 1]
```diff
@@ -91,11 +91,11 @@
       ["First-Cell"          kotl-mode:beginning-of-buffer  t]
       ["Last-Cell"           kotl-mode:end-of-buffer        t])
     '("Label-Type"
-      ["Alphanumeric (Default)"  (kview:set-label-type kview 'alpha)  t]
-      ["Legal"                   (kview:set-label-type kview 'legal)  t]
```
Comment (rswgnu) [inline kotl/kmenu.el:95]: Shouldn’t we have gotten unbound dynamic variable warnings here?

**Review by rswgnu — APPROVED** [review]
See my inline question.

File: kotl/kmenu.el (hunk 1 above)
Comment (matsl) [reply kotl/kmenu.el:95]: Yes, at a first look you would think that. Can it be that `INITVALUE` of the defconst is just an s-exp with no real meaning until it is used as a menu? 

The `kmenu.el` file is free of requiring any packages that define functions used in the menus.

Comment (rswgnu) [reply kotl/kmenu.el:95]: Yes, the menu expressions are quoted, so not evaluated until used, so I would just expect these to fail when used and therefore need testing to see if that were true prior to applying the suggested fix.

**Pull request merged**